readme.md

```diff
@@ -276,18 +276,18 @@ The `targets` array is to contain a list of entities (to be scraped from) keyed
 - `dissect`: Split compound attributes into smaller attributes via REGEX. (optional)
 - `preprocess`: Array of attributes which need to be preprocessed. `[attributeName => callable]` (optional)
 - `remap`: Array of attributes which need to be renamed in order to be saved as target objects. `[attributeName => newName]` (optional)
-- `bad_words`: Any scraps found containing these words will be discarded. (optional)
+- `bad_words`: Any scrapes found containing these words will be discarded. (optional)
 
 ## Glossary of Terms
 The following words may appear in context above.
 
-- `Daemon`: User instance to be used by the scavenger service.
+- `Daemon`: User instance to be used by the Scavenger service.
 - `Scrap`: Scraped data before being converted to the target object.
 - `Target`: Configured source-model mapping for a single entity. 
-- `Target Object`: Eloquent model object to be generated from scrap. 
+- `Target Object`: Eloquent model object to be generated from scrape. 
```

Comment:
```suggestion
- `Target Object`: Eloquent model object to be generated from scrap. 
```

This is referring to a "scrap", as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

Reply:
But "scrap" is "a scrap of paper".  And "scrape" is "I am going to scrape that webpage"

Reply:
Makes sense, however a "scrap" is defined in this context as:

`Scraped data before being converted to the target object.`

Reply:
Sure, but that's a failure in naming, and not the fault of the README. It really should be `scrape`. However, I won't push the point any further, and accept your suggestions.